activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -64,6 +60,11 @@ def write_entry(key, value, options)
           def delete_entry(key, options)
             !!@data.delete(key)
           end
+
+          private
+            def fetch_entry(key, options = nil) # :nodoc:
```

Comment:
Wups! Missed that this wasn't called with an implicit self — it should be `protected` not `private` :smile: